src/main/kotlin/it/pagopa/ecommerce/payment/requests/repositories/PaymentRequestInfo.kt

```diff
@@ -6,7 +6,7 @@ import org.springframework.data.annotation.Id
 import org.springframework.data.annotation.PersistenceCreator
 import org.springframework.data.redis.core.RedisHash
 
-@RedisHash(value = "keys", timeToLive = 10 * 60)
+@RedisHash(value = "keys", timeToLive = 60)
```

Comment:
This value is not used since the switch to RedisTemplate. TTL can be configured here https://github.com/pagopa/pagopa-ecommerce-payment-requests-service/blob/8fc4bc50cedfe6d2066acb643956b9840306c7a5/src/main/kotlin/it/pagopa/ecommerce/payment/requests/configurations/redis/RedisConfiguration.kt#L36